### ISSUE TITLE
Build error with 2.6.0

### ISSUE THREAD

**Issue by ih57452** [RaphaelJenni/FirebaseUI-Angular#32]
I'm getting this error when I try to build with version 2.6.0:
```
ERROR in ./node_modules/firebaseui-angular/index.js
Module not found: Error: Can't resolve '@firebase/auth-types' in 'C:\Users\Bryan\Documents\GitHub\woodstock\node_modules\firebaseui-angular'
```
I installed `@firebase/auth-types` in my main `./node_modules` folder but it still doesn't work. Should it be included in the peer dependencies?

**Comment by RaphaelJenni** [comment]
Try to update firebase to the newest version.

**Comment by ih57452** [comment]
I have firebase 4.9.1 and the newest versions of everything else installed. I went to the `firebaseui-angular/sample`  folder and ran `npm install` and `ng build` and I get the same error. Here is the full error text:
```
ERROR in ./node_modules/firebaseui-angular/index.js
Module not found: Error: Can't resolve '@firebase/auth-types' in 'C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\node_modules\firebaseui-angular'
resolve '@firebase/auth-types' in 'C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\node_modules\firebaseui-angular'
  Parsed request is a module
  using description file: C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\node_modules\firebaseui-angular\package.json (relative path: .)
    Field 'browser' doesn't contain a valid alias configuration
  after using description file: C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\node_modules\firebaseui-angular\package.json (relative path: .)
    resolve as module
      C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\node_modules\firebaseui-angular\node_modules doesn't exist or is not a directory
      C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\node_modules\node_modules doesn't exist or is not a directory
      C:\Users\Bryan\Documents\GitHub\node_modules doesn't exist or is not a directory
      C:\Users\Bryan\Documents\node_modules doesn't exist or is not a directory
      C:\Users\node_modules doesn't exist or is not a directory
      C:\node_modules doesn't exist or is not a directory
      looking for modules in C:\Users\Bryan\node_modules
        No description file found
        Field 'browser' doesn't contain a valid alias configuration
        No description file found
        no extension
          Field 'browser' doesn't contain a valid alias configuration
          C:\Users\Bryan\node_modules\@firebase\auth-types doesn't exist
        .ts
          Field 'browser' doesn't contain a valid alias configuration
          C:\Users\Bryan\node_modules\@firebase\auth-types.ts doesn't exist
        .js
          Field 'browser' doesn't contain a valid alias configuration
          C:\Users\Bryan\node_modules\@firebase\auth-types.js doesn't exist
        as directory
          C:\Users\Bryan\node_modules\@firebase\auth-types doesn't exist
      looking for modules in C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\node_modules
        using description file: C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\package.json (relative path: ./node_modules)
          Field 'browser' doesn't contain a valid alias configuration
        after using description file: C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\package.json (relative path: ./node_modules)
          using description file: C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\node_modules\@firebase\auth-types\package.json (relative path: .)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\node_modules\@firebase\auth-types is not a file
            .ts
              Field 'browser' doesn't contain a valid alias configuration
              C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\node_modules\@firebase\auth-types.ts doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\node_modules\@firebase\auth-types.js doesn't exist
            as directory
              existing directory
                using path: C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\node_modules\@firebase\auth-types\index
                  using description file: C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\node_modules\@firebase\auth-types\package.json (relative path: ./index)
                    no extension
                      Field 'browser' doesn't contain a valid alias configuration
                      C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\node_modules\@firebase\auth-types\index doesn't exist
                    .ts
                      Field 'browser' doesn't contain a valid alias configuration
                      C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\node_modules\@firebase\auth-types\index.ts doesn't exist
                    .js
                      Field 'browser' doesn't contain a valid alias configuration
                      C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\node_modules\@firebase\auth-types\index.js doesn't exist
      looking for modules in C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\node_modules        using description file: C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\package.json (relative path: ./node_modules)
          Field 'browser' doesn't contain a valid alias configuration
        after using description file: C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\package.json (relative path: ./node_modules)
          using description file: C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\node_modules\@firebase\auth-types\package.json (relative path: .)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\node_modules\@firebase\auth-types is not a file
            .ts
              Field 'browser' doesn't contain a valid alias configuration
              C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\node_modules\@firebase\auth-types.ts doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\node_modules\@firebase\auth-types.js doesn't exist
            as directory
              existing directory
                using path: C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\node_modules\@firebase\auth-types\index
                  using description file: C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\node_modules\@firebase\auth-types\package.json (relative path: ./index)
                    no extension
                      Field 'browser' doesn't contain a valid alias configuration
                      C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\node_modules\@firebase\auth-types\index doesn't exist
                    .ts
                      Field 'browser' doesn't contain a valid alias configuration
                      C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\node_modules\@firebase\auth-types\index.ts doesn't exist
                    .js
                      Field 'browser' doesn't contain a valid alias configuration
                      C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\node_modules\@firebase\auth-types\index.js doesn't exist
      looking for modules in C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\node_modules
        using description file: C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\package.json (relative path: ./node_modules)
          Field 'browser' doesn't contain a valid alias configuration
        after using description file: C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\package.json (relative path: ./node_modules)
          using description file: C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\node_modules\@firebase\auth-types\package.json (relative path: .)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\node_modules\@firebase\auth-types is not a file
            .ts
              Field 'browser' doesn't contain a valid alias configuration
              C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\node_modules\@firebase\auth-types.ts doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\node_modules\@firebase\auth-types.js doesn't exist
            as directory
              existing directory
                using path: C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\node_modules\@firebase\auth-types\index
                  using description file: C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\node_modules\@firebase\auth-types\package.json (relative path: ./index)
                    no extension
                      Field 'browser' doesn't contain a valid alias configuration
                      C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\node_modules\@firebase\auth-types\index doesn't exist
                    .ts
                      Field 'browser' doesn't contain a valid alias configuration
                      C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\node_modules\@firebase\auth-types\index.ts doesn't exist
                    .js
                      Field 'browser' doesn't contain a valid alias configuration
                      C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\node_modules\@firebase\auth-types\index.js doesn't exist
      looking for modules in C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\src
        using description file: C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\package.json (relative path: ./src)
          Field 'browser' doesn't contain a valid alias configuration
        after using description file: C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\package.json (relative path: ./src)
          using description file: C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\package.json (relative path: ./src/@firebase/auth-types)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\src\@firebase\auth-types doesn't exist
            .ts
              Field 'browser' doesn't contain a valid alias configuration
              C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\src\@firebase\auth-types.ts doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\src\@firebase\auth-types.js doesn't exist
            as directory
              C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\src\@firebase\auth-types doesn't exist
      looking for modules in C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\src
        using description file: C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\package.json (relative path: ./src)
          Field 'browser' doesn't contain a valid alias configuration
        after using description file: C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\package.json (relative path: ./src)
          using description file: C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\package.json (relative path: ./src/@firebase/auth-types)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\src\@firebase\auth-types doesn't exist
            .ts
              Field 'browser' doesn't contain a valid alias configuration
              C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\src\@firebase\auth-types.ts doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\src\@firebase\auth-types.js doesn't exist
            as directory
              C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\src\@firebase\auth-types doesn't exist
[C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\node_modules\firebaseui-angular\node_modules]
[C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\node_modules\node_modules]
[C:\Users\Bryan\Documents\GitHub\node_modules]
[C:\Users\Bryan\Documents\node_modules]
[C:\Users\node_modules]
[C:\node_modules]
[C:\Users\Bryan\node_modules\package.json]
[C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\node_modules\@firebase\auth-types]
[C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\node_modules\@firebase\auth-types]
[C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\node_modules\@firebase\auth-types.ts]
[C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\node_modules\@firebase\auth-types.ts]
[C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\node_modules\@firebase\auth-types.js]
[C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\node_modules\@firebase\auth-types.js]
[C:\Users\Bryan\node_modules\@firebase\auth-types\package.json]
[C:\Users\Bryan\node_modules\@firebase\auth-types]
[C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\node_modules\@firebase\auth-types\index]
[C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\node_modules\@firebase\auth-types\index][C:\Users\Bryan\node_modules\@firebase\auth-types.ts]
[C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\node_modules\@firebase\auth-types\index.ts]
[C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\node_modules\@firebase\auth-types\index.ts]
[C:\Users\Bryan\node_modules\@firebase\auth-types.js]
[C:\Users\Bryan\node_modules\@firebase\auth-types]
[C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\node_modules\@firebase\auth-types\index.js]
[C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\node_modules\@firebase\auth-types\index.js]
[C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\node_modules\@firebase\auth-types]
[C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\node_modules\@firebase\auth-types.ts]
[C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\node_modules\@firebase\auth-types.js]
[C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\node_modules\@firebase\auth-types\index]
[C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\node_modules\@firebase\auth-types\index.ts]
[C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\node_modules\@firebase\auth-types\index.js]
[C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\src\@firebase\auth-types]
[C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\src\@firebase\auth-types.ts]
[C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\src\@firebase\auth-types.js]
[C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\src\@firebase\auth-types]
[C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\src\@firebase\auth-types]
[C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\src\@firebase\auth-types.ts]
[C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\src\@firebase\auth-types.js]
[C:\Users\Bryan\Documents\GitHub\FirebaseUI-Angular\sample\src\@firebase\auth-types]
 @ ./node_modules/firebaseui-angular/index.js 5:0-159
 @ ./src/app/app.module.ts
 @ ./src/main.ts
 @ multi ./src/main.ts
```
When I searched for `Field 'browser' doesn't contain a valid alias configuration` I found a couple of bugs on the Angular CLI, but I don't know if they're causing this or not.

**Comment by RaphaelJenni** [comment]
Got the same error. I'll look into it.

**Comment by RaphaelJenni** [comment]
Should be fixed in version 2.6.1
Please check it and close the issue if it works

**Comment by ih57452** [comment]
Works fine with 2.6.1. Thanks!
